src/dotnet/Fable.Core/Import/Node/Url.fs

```diff
@@ -0,0 +1,28 @@
+module rec Fable.Import.Node.Url
+
+open Fable.Core
+
+type [<AllowNullLiteral>] Url<'a> =
+    abstract href: string option with get, set
+    abstract protocol: string option with get, set
+    abstract auth: string option with get, set
+    abstract hostname: string option with get, set
+    abstract port: string option with get, set
+    abstract host: string option with get, set
+    abstract pathname: string option with get, set
+    abstract search: string option with get, set
+    abstract query: 'a option with get, set
+    abstract slashes: bool option with get, set
+    abstract hash: string option with get, set
+    abstract path: string option with get, set
+
+type Globals =
+    abstract parse: urlStr: string -> Url<string>
+    abstract parse: urlStr: string * ?parseQueryString: bool -> Url<obj>
+    abstract parse: urlStr: string * ?parseQueryString: bool * ?slashesDenoteHost: bool -> Url<U2<string, obj>>
+    abstract format: url: Url<string> -> string
+    abstract format: url: Url<obj> -> string
+    abstract resolve: from: string * ``to``: string -> string
+
+[<Import("*","url")>]
+let exports: Globals = jsNative
```

Comment:
This will be really hinder the readability of code because of having to write `Url.exports.someting` or `Querystring.exports.somethingElse` everytime. someone coming from Node expects `url.something` and `queryString.somethingElse` which are a lot more logical given that "exports" is a way to originize js modules not F# modules